### DISCUSSION TITLE
Samsung S24: OwnTracks stops working after one day

### DISCUSSION THREAD

**Discussion by SiggiKeller** [owntracks/android#1887] [Q&A]
I had a Galaxy S22+ before and OT worked quite well. Now I changed to a new Galaxy S24. OT works for a day and than stops working until I manually open the app. Obviously opening the app wakes it up and it starts working again.
I did change all possible options in the phone around energy saving etc. I also put OwnTracks in the list of apps, that never are forced in standby. But nothing helps.
BTW OT runs in significant change mode. Now I'll test it in the moving mode (don't know how this is called correctly).

Rgds,
Siggi

**Comment by jpmens** [comment]
Please let us know which version of OwnTracks (and Android) you are running, and where you downloaded it from.

**Reply by SiggiKeller** [reply]
The version is 2.5.3 (420503003) from Google Play. Android is version 14.

**Reply by growse** [reply]
Sadly, Samsung devices are notorious for unilaterally killing foreground services. 

https://dontkillmyapp.com/

**Reply by SiggiKeller** [reply]
I tried all different options and settings. Nothing seems to help. The app runs, but does not send location updates as defined in the settings. Today I tried the movement mode with 10 s update interval. It sent some updates from time to time but far less than every 10 s.
I‘m giving up!

**Comment by xep22** [comment]
Same for OnePlus 12.
